custom.js

```diff
@@ -12,16 +12,16 @@ function init (type, message, cause) {
 }
 
 // generic prototype, not intended to be actually used - helpful for `instanceof`
-function CustomError (message, cause) {
+var CustomError = function f(message, cause) {
   Error.call(this)
   if (Error.captureStackTrace)
-    Error.captureStackTrace(this, arguments.callee)
+    Error.captureStackTrace(this, f)
   init.call(this, 'CustomError', message, cause)
 }
 
 CustomError.prototype = new Error()
 
-function createError (errno, type, proto) {
+var createError = function f(errno, type, proto) {
   var err = function (message, cause) {
```

Comment:
Actually, I think it's more correct to use the inner anonymous function instead, since it's actually part of the callstack when the error is created (i.e. not when the error _type_ is created).